src/components/SearchHeader.vue

```diff
@@ -55,6 +55,10 @@ export default {
       type: String,
       required: true,
     },
+    country: {
+      type: String,
+      required: false,
```

Comment:
Ik zie dat deze prop in de `SearchDetail` wel verplicht is maar hier niet. Is dat bewust?

Reply:
Ik twijfelde eigenlijk of ik hem verplicht moest maken of niet en die twijfel is hier dus te zien 😅